apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx

```diff
@@ -35,7 +35,7 @@ If you're one of those contributors, thank you!
 - [`storage-js`](https://github.com/supabase/storage-js/graphs/contributors) (17)
 - [`supabase-js`](https://github.com/supabase/supabase-js/graphs/contributors) (39)
 
-Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/pixtron), and [@karlseguin](https://github.com/karlseguin).
+Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/Acknowledgements), and [@karlseguin](https://github.com/karlseguin).
```

Comment:
should be https://github.com/bnjmnt4n instead? https://github.com/Acknowledgements is a empty profile

Reply:
Wow, all the mistakes that I make. How did I do that😂
```suggestion
Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@pixtron](https://github.com/Acknowledgements), and [@karlseguin](https://github.com/karlseguin).
```